source/beamline/CLS/CLSXIAFiltersControl.h

```diff
@@ -0,0 +1,88 @@
+#ifndef CLSXIAFILTERSCONTROL_H
+#define CLSXIAFILTERSCONTROL_H
+
+#include "beamline/AMPseudoMotorControl.h"
+#include "beamline/AMPVControl.h"
+
+#include <QMap>
+
+class CLSXIAFiltersControl : public AMPseudoMotorControl
```

Comment:
You don't need to redeclare the pure virtual methods inside of subclasses (e.g.: createMoveAction, updateConnected, etc.).